crates/sui-protocol-config/src/lib.rs

```diff
@@ -581,6 +581,11 @@ struct FeatureFlags {
     // Enable v2 native charging for natives.
     #[serde(skip_serializing_if = "is_false")]
     native_charging_v2: bool,
+
+    // Enable support for ConsensusV2 objects.
+    // WARNING: This feature is experimental and incomplete.
+    #[serde(skip_serializing_if = "is_false")]
+    consensus_v2_objects: bool,
```

Comment:
i think its worth having a dedicated flag for the new shared object tables, and then a new one that actually enables the consensus_v2_objects feature later on. Overloading flags can cause a lot of pain

Reply:
Also - do we need a protocol flag to choose the correct table? i think this can be done with a epoch flag since the new tables implement a superset of the functionality of the old tables.

this will make it much clearer that the old code can be deleted in the future once the migration is complete